CONTRIBUTING.md

```diff
@@ -109,6 +109,25 @@ The code is checked for type errors by [TypeScript](https://www.typescriptlang.o
   ```
 - VS Code will also run type-checking while editing source code, providing immediate feedback.
 
+#### Changing TypeScript Version in VS Code's Command Palette
+
+For TypeScript to work properly in the Monorepo the version used in VSCode must be the project's current TypeScript version, follow these steps:
```

Comment:
Should/can we just enforce this with a `.vscode/settings.json` file in the repo?

Reply:
Doesn't seem to work in this branch: https://github.com/cloudflare/workers-sdk/pull/1042

Reply:
We can look into that more too.